x-pack/legacy/plugins/reporting/server/lib/esqueue/worker.js

```diff
@@ -312,7 +315,11 @@ export class Worker extends events.EventEmitter {
       this.warn(`Failure occurred on job ${job._id}`, jobErr);
       this.emit(constants.EVENT_WORKER_JOB_EXECUTION_ERROR, this._formatErrorParams(jobErr, job));
       return this._failJob(job, (jobErr.toString) ? jobErr.toString() : false);
-    });
+    })
+      .then((response) => {
+        const formattedDocPath = `/${response._index}/${response._type}/${response._id}`;
+        this.info(`Job data saved successfully: ${formattedDocPath}`);
+      });
```

Comment:
Oops, this needs to go ABOVE the error catching

Reply:
https://github.com/elastic/kibana/pull/43241/commits/7a1734866e92939ca8a0db0901a96ec319664a18